easybuild/easyblocks/m/mesa.py

```diff
@@ -179,3 +179,12 @@ def sanity_check_step(self):
             custom_paths['files'].extend(swr_arch_libs)
 
         super(EB_Mesa, self).sanity_check_step(custom_paths=custom_paths)
+
+    def make_module_extra(self, *args, **kwargs):
+        """ Append to EGL vendor library path,
+        so that any NVidia libraries take precedence. """
+        txt = ''
+        # Append rather than prepend path to ensure that system NVidia drivers have priority.
+        txt += self.module_generator.append_paths('__EGL_VENDOR_LIBRARY_DIRS', 'share/glvnd/egl_vendor.d')
+        txt += super(EB_Mesa, self).make_module_extra(*args, **kwargs)
```

Comment:
```suggestion
        txt = super(EB_Mesa, self).make_module_extra(*args, **kwargs)
        # Append rather than prepend path to ensure that system NVidia drivers have priority.
        txt += self.module_generator.append_paths('__EGL_VENDOR_LIBRARY_DIRS', 'share/glvnd/egl_vendor.d')
```